test/OpenTelemetry.Exporter.Geneva.Tests/ConnectionStringBuilderTests.cs

```diff
@@ -233,5 +233,7 @@ public void ConnectionStringBuilder_PrivatePreviewEnableUserEvents_No_Default_Va
         Assert.True(builder.PrivatePreviewEnableUserEvents);
         builder = new ConnectionStringBuilder("PrivatePreviewEnableUserEvents=false");
         Assert.False(builder.PrivatePreviewEnableUserEvents);
+        builder = new ConnectionStringBuilder("PrivatePreviewEnableAFDCorrelationIdEnrichment=false");
+        Assert.False(builder.PrivatePreviewEnableAFDCorrelationIdEnrichment);
```

Comment:
This feature is opt-in, correct?
So the Assert should be false even without the change on line 236?